Dockerfile.tests

```diff
@@ -9,8 +9,11 @@ FROM salus-local
 
 COPY spec /home/spec
 
+# Symlink for gosec testing
+RUN ln -s /home/spec $GOPATH/src/repo/spec
+
 RUN bundle install --with test
 
-RUN rm ./spec/docker/docker_spec.rb && rm ./spec/static_analysis/rubocop_spec.rb
+RUN rm ./spec/docker/docker_spec.rb
```

Comment:
`rm ./spec/static_analysis/rubocop_spec.rb` kept failing stating file doesn't exist, I can add it back in if necessary.

Reply:
Hmmm yeah that doesn't exist anymore thanks to the new CircleCI flow. Isn't this container built in the tests? How come this was only noticed now?

Reply:
not sure, it happened as I was building the container locally. Could the image settings be different on circleci?

Reply:
I find the Dockerfile.tests really useful as well. Noting this as an improvement.